src/test/java/teammates/test/cases/common/SanitizerTest.java

```diff
@@ -44,27 +47,62 @@ public void testSanitizeTextField() {
     
     @Test
     public void testSanitizeForJs() {
-        
+
```

Comment:
Unnecessary blank lines after a method - let's remove all these to keep it slightly shorter